### ISSUE TITLE
trainer文件中的代码疑问

### ISSUE THREAD

**Issue by husterWL** [YeexiaoZheng/Multimodal-Sentiment-Analysis#5]
pred, loss = self.model(texts, texts_mask, imgs, labels=labels)以及下面的几个同样调用model实例对象的代码，好像并没有用forward()函数，而且各个模型的代码中Model类中并没有声明__call__函数。

**Comment by YeexiaoZheng** [comment]
有调用的 model实例对象是在每个Model.py的最后一个模型叫Model的 这个Model有forward函数 也是这个self.model(texts, texts_mask, imgs, labels=labels)调用的函数
![image](https://github.com/YeexiaoZheng/Multimodal-Sentiment-Analysis/assets/97672942/88643779-5195-4308-be95-7b5302a94a2c)

**Comment by husterWL** [comment]
这个forward函数我是知道的，我只是好奇为什么不写成pred, loss = self.model.forward(texts, texts_mask, imgs, labels=labels)

**Comment by YeexiaoZheng** [comment]
哦哦 因为这个是pytorch框架的简写写法 默认__call__函数调用的就是forward()

**Comment by husterWL** [comment]
ok明白了，非常感谢！
